plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -189,12 +189,35 @@ chain_plugin::chain_plugin()
    app().register_config_type<eosio::chain::db_read_mode>();
    app().register_config_type<eosio::chain::validation_mode>();
    app().register_config_type<chainbase::pinnable_mapped_file::map_mode>();
+   app().register_config_type<eosio::chain::wasm_interface::vm_type>();
 }
 
 chain_plugin::~chain_plugin(){}
 
 void chain_plugin::set_program_options(options_description& cli, options_description& cfg)
 {
+   // build wasm_runtime help text
```

Comment:
in 2.0 there is another run time, wabt. We'll need to come up with proper description for it :innocent:

Reply:
changed it as wabt.
"wabt" : The WebAssembly Binary Toolkit.